docs/absolute.md

```diff
@@ -319,6 +327,10 @@ This is because months and years can be different lengths depending on which mon
 You cannot determine the start and end date of a difference between `Temporal.Absolute`s, because `Temporal.Absolute` has no time zone or calendar.
 In addition, weeks can be different lengths in different calendars, and days can be different lengths when the time zone has a daylight saving transition.
 
+You can round the result using the `smallestUnit`, `roundingIncrement`, and `roundingMode` options.
+These behave as in the `Temporal.Duration.round()` method, but increments of days and larger are counted relative to `absolute`.
```

Comment:
Could you explain this sentence?  I don't fully understand what you're trying to convey.

Reply:
Oops, this didn't apply to Absolute rounding anyway. In DateTime I'll change it to "These behave as in the `Temporal.Duration.round()` method, but increments of days and larger are allowed.
Since rounding to an increment expressed in days or larger units requires a reference point, `dateTime` is used as the reference point in that case."

Reply:
Cool. 

> `dateTime` is used as the reference point in that case."

Shouldn't it be this instead?

>  `this` is used as the reference point in that case

Reply:
I guess that's a matter of writing style; elsewhere in the docs we refer to the `this` object as whatever it's called in the `<h3>`, as in "**Returns:** a `Temporal.Duration` representing the difference between `datetime` and `other`." (rather than "...`this` and `other`")

Reply:
Personally I find `this` to be clearer (and easier for copy/pasting into different .md files!) but I don't have a strong opinion one way or the other.  Either way is fine.

Reply:
I'm not sure how it got started this way, tbh - I checked on MDN and it seems like they actually avoid using either way, which actually seems worse to me.